src/proposals/20231115/SparkGnosis_20231115.t.sol

```diff
@@ -12,84 +12,10 @@ contract SparkGnosis_20231115Test is SparkGnosisTestBase {
     }
 
     function setUp() public {
-        vm.createSelectFork(getChain('gnosis_chain').rpcUrl);
-        payload = deployPayload();
+        vm.createSelectFork(getChain('gnosis_chain').rpcUrl, 30_855_380);
```

Comment:
Small nit, but can we start forking from the exact block of the spell being deployed?